src/Marker.tsx

```diff
@@ -90,6 +93,7 @@ export class Marker extends React.Component<MarkerProps, object> {
     return null;
   }
 
+  // Note: Only call this when map is already mounted.
```

Comment:
It's not that important, but it's better here to check if map exists and throw an error if it doesn't, that way it's obvious what is wrong instead of trying to debug errors later on